index.js

```diff
@@ -155,6 +155,10 @@ const isAndroid = () => {
 }
 
 /** @private **/
+const isKaiOS = () => {
+  const ua = (scope.navigator || {}).userAgent || ''
```

Comment:
I was googeling for it, but could not find any other option. The problem was, if it is not marked as `isFirefox`, then it calls `resumeInfinity(utterance)` in `utterance.addEventListener('start')`, from which it never recovers. I will search for another option, but at the moment, I think, this is the only option :see_no_evil: ...

Reply:
Yeah, I tried to get deeper into their stack but it seems to be not that well documented. Let's use the UA for now but keep in mind that this may not be 100% compatible.